openapi/schemas/Roles.yaml

```diff
@@ -302,6 +302,55 @@ NewCandidate:
       type: string
       example: "https://voto.vote"
 
+NewCandidateLight:
+  type: object
+  required:
+    - gender_id
+    - party_id
+    - firstName
+    - lastName
+  properties:
+    gender_id:
+      type: integer
+      example: 123
+    party_id:
+      type: integer
+      example: 123
+    firstName: 
+      type: string
+      example: "Max"
+      description: "Firstname of candidate light. Can also contain title as prefix"
+    lastName: 
+      type: string
+      example: "Max"
+      description: "Firstname of candidate light. Can also contain title as prefix"
```

Comment:
Should be Lastname of candidate light. (No prefix)

Reply:
Thanks for the review, fix is pushed.